src/config/config-development.ts

```diff
@@ -1,20 +1,6 @@
 import { Container } from 'inversify';
-
-import { jwtOptionsInjectSymbol } from '../authentication';
-import { adminTeamNameInjectSymbol, openTeamNamesInjectSymbol } from '../shared/types';
 import productionConfig from './config-production';
-import { getJwtOptions } from './config-test';
```

Comment:
`getJwtOptions` is now not used anywhere else other than `config-test`, so there's no need to export it anymore.